### ISSUE TITLE
puma can't start for environment problem

### ISSUE THREAD

**Issue by chendelin1982** [Websoft9/ansible-redmine#2]
Start Redmine by `puma -b tcp://127.0.0.1:9292 -e production` is OK, but use systemd, puma can't start for enviroment problem

**Comment by chendelin1982** [comment]
refer to: https://unix.stackexchange.com/questions/347873/set-path-for-a-systemd-unit

**Comment by zengxc-1996** [comment]
```shell
Dec 30 09:20:20 iZj6c56tahdu693emhjntfZ mysqld[1598]: 2020-12-30T01:20:20.731363Z 15 [Note] Access denied for user 'redmine'@'localhost' (using password: YES)
Dec 30 09:20:32 iZj6c56tahdu693emhjntfZ mysqld[1598]: 2020-12-30T01:20:32.239089Z 16 [Warning] IP address '172.18.0.2' could not be resolved: Name or service not known
Dec 30 09:23:07 iZj6c56tahdu693emhjntfZ mysqld[1598]: 2020-12-30T01:23:07.259595Z 38 [Note] Access denied for user 'redmine'@'localhost' (using password: YES)
Dec 30 09:25:50 iZj6c56tahdu693emhjntfZ sshd[2366]: Connection closed by 192.241.221.137 port 48844 [preauth]
```
When I execute `systemctl status redmine`

**Comment by zengxc-1996** [comment]
Use bash shell to source /etc/profile.d and run puma
```
ExecStart=/bin/bash -c 'source /etc/profile.d/rvm.sh;puma -b tcp://127.0.0.1:9292 -e production' 
```
Done
